pkg/function/export_rds_snapshot_location.go

```diff
@@ -223,20 +267,45 @@ func postgresBackupCommand(instanceID, dbEndpoint, username, password, backupPre
 		"-c",
 		fmt.Sprintf(`
 			export PGHOST=%s
-			export PGUSER=%s
-			export PGPASSWORD=%s
 			BACKUP_PREFIX=%s
 			BACKUP_ID=%s
 
 			mkdir /backup
-			restricted=("template0", "rdsadmin")
-			psql -lqt | awk -F "|" '{print $1}' | tr -d " " | sed '/^$/d' |
-			while read db;
-			  do [[ ! ${restricted[@]} =~ ${db} ]] && echo "backing up $db db" && pg_dump $db -C > /backup/$db.sql;
+			dblist=( %s )
+			for db in "${dblist[@]}";
+			  do echo "backing up $db db" && pg_dump $db -C > /backup/$db.sql;
```

Comment:
Check if pg_dump cli supports multiple dbs

Reply:
@tdmanv it is not possible to take backup of multiple databases with pg_dump command. https://www.postgresql.org/docs/current/app-pgdump.html
```
pg_dump only dumps a single database. To back up an entire cluster, or to back up global objects that are common to all databases in a cluster (such as roles and tablespaces), use pg_dumpall.
```